paymentsheet/src/main/java/com/stripe/android/customersheet/ui/CustomerSheetScreen.kt

```diff
@@ -181,3 +195,58 @@ internal fun AddCard(
         )
     }
 }
+
+@Composable
+internal fun AddPaymentMethodWithPaymentElement(
+    viewState: CustomerSheetViewState.AddPaymentMethod,
+    viewActionHandler: (CustomerSheetViewAction) -> Unit,
+    formViewModelSubComponentBuilderProvider: Provider<FormViewModelSubcomponent.Builder>?,
+) {
+    requireNotNull(formViewModelSubComponentBuilderProvider)
+    val horizontalPadding = dimensionResource(R.dimen.stripe_paymentsheet_outer_spacing_horizontal)
+
+    Column {
+        H4Text(
+            text = stringResource(id = R.string.stripe_paymentsheet_save_a_new_payment_method),
+            modifier = Modifier
+                .padding(bottom = 20.dp)
```

Comment:
Spacing looks rather large. Can we align this with PaymentSheet?

Reply:
I will make sure to do this in a future PR!